.circleci/config.yml

```diff
@@ -0,0 +1,56 @@
+aliases:
+  - &job-defaults
+    working_directory: ~/repo
+    docker:
+      - image: circleci/node:10
```

Comment:
would be great to have jobs for node 8, 10 and 11 (I guess those are mostly in use)

Reply:
Oh right, haven't discussed what node versions we support. Will research!

Reply:
here is a sample just in case: https://github.com/castle/c.js/blob/master/.circleci/config.yml